lib/pages/chat/chat.dart

```diff
@@ -1279,6 +1280,8 @@ class ChatController extends State<Chat>
     }
   }
 
+  final CancelToken mediaCancelToken = CancelToken();
```

Comment:
i think it may not work well in case of multiple images get cancel, try send multiple images

Reply:
we should have the better way for the case of multiple uploading